CHANGELOG.md

```diff
@@ -1,27 +1,46 @@
-# CHANGELOG 
+# CHANGELOG
+
+## v3.4.0
+### Added
+-[\#745](https://github.com/Manta-Network/Manta/pull/745) Workflow to check for labels
```

Comment:
Add one space between `-` and `[`, or the markdown won't be parsed correctly.

Reply:
https://github.com/Manta-Network/dev-tools/pull/12 fixed it in the generator itself

Reply:
Pushed the new changelog. It renders fine. need to merge https://github.com/Manta-Network/dev-tools/pull/12 for verification to succeed